events/2016/1-nodeschool.md

```diff
@@ -1,7 +1,8 @@
 ---
 name: Nodeschool
+slug: nodeschool
 date: 2016-06-11
-info: http://www.meetup.com/JSOxford/events/231241282/
+meetup: http://www.meetup.com/JSOxford/events/231241282/
```

Comment:
Could we make this attribute more generic in case we have an external event which is ticketed by eventbrite or tito?

Something like `tickets_url`?